packages/@o3r/localization/schematics/localization-to-component/schema.ts

```diff
@@ -0,0 +1,15 @@
+import type { JsonObject } from '@angular-devkit/core';
+
+export interface NgAddLocalizationSchematicsSchema extends JsonObject {
+  /** Path to the component */
+  path: string;
+
+  /** Path to spec file of the component */
+  specFilePath: string;
```

Comment:
is this used?

Reply:
https://github.com/AmadeusITGroup/otter/pull/464/files#diff-c7ba2237ccab109059dc14110a7840ecc6761ba6ec20147eeaa9a421768de162R296